Content.Server/Nutrition/EntitySystems/SliceableFoodSystem.cs

```diff
@@ -65,6 +70,12 @@ private void OnInteractUsing(Entity<SliceableFoodComponent> entity, ref Interact
             FillSlice(sliceUid, lostSolution);
 
             _audio.PlayPvs(component.Sound, transform.Coordinates, AudioParams.Default.WithVolume(-2));
+
+            // This isn't an inpluse because cuts have very different densities making some get thrown super far..
+            var randVect = _random.NextVector2(2.0f, 2.5f);
+            if (TryComp<PhysicsComponent>(sliceUid, out var physics))
+                _physics.SetLinearVelocity(sliceUid, randVect, body: physics);
```

Comment:
this really just ought to be done with the random offset method for the spawn location of the item. applying physics impulses is kinda weird and very small offsets like this aren't going to clip into anything anyways.

Reply:
OK I did it but honestly I don't like it as much! The slices moving around is more fun even if it is cartoonie and less "realistic".

Obviously its up to you and if the PR wont get merged with the velocity I'll change it as its better than before even without velocity.

https://github.com/space-wizards/space-station-14/assets/107373427/ac8fa26c-f6a9-46ef-b427-fe636c70d77b

Reply:
Yeah I asked in the discord and it seems to be split 50/50 on which one is better...

Reply:
I tried asking on discord a while ago but got no response! I'm just going to resolve this for now (I like the physics version more!) but if this wont get merged without doing the random offset let me know and I'll change it 👍